debug/test/browser/devtools.test.js

```diff
@@ -137,790 +137,797 @@ function getRoot(element) {
 	return element._children;
 }
 
-describe('devtools', () => {
+let ua = window.navigator.userAgent;
+if (
+	ua.indexOf('Edge') === -1 &&
+	ua.indexOf('Chrome') > -1 &&
+	ua.indexOf('Firefox') > -1
+) {
```

Comment:
This `if`-statement is the only change here in this file. Everything else is just indentation. I tried using the `this.skip`-API like the mocha docs recommend, but it will still run some tests contrary to what the docs say.

Reply:
could you like exit or return or something for early exit instead of indenting everything?

Reply:
I believe this works:

```js
const desc = /Chrome|Firefox/.test(navigator.userAgent) && !/Edge/.test(navigator.userAgent) ? describe : describe.skip;

desc('..', () => {
  ..
```

Reply:
That's clever! I like it :+1:

Reply:
@ForsakenHarmony tried the early return, and it works too. I think I'll go with @developit 's approach though, it's more clear and prints that the `devtools` test have been skipped :slightly_smiling_face:

Reply:
@ForsakenHarmony I was wrong! It doesn't make any difference :joy: